src/dotnet-publish-iis/WebConfigTransform.cs

```diff
@@ -64,6 +64,25 @@ private static void TransformHttpPlatform(XElement httpPlatformElement, string a
             SetAttributeValueIfEmpty(httpPlatformElement, "stdoutLogEnabled", "false");
             SetAttributeValueIfEmpty(httpPlatformElement, "stdoutLogFile", logPath);
             SetAttributeValueIfEmpty(httpPlatformElement, "startupTimeLimit", "3600");
+
+            AddApplicationBase(httpPlatformElement);
+        }
+
+        private static void AddApplicationBase(XElement httpPlatformElement)
+        {
+            const string appBaseKeyName = "ASPNET_APPLICATIONBASE";
+
+            var envVariables = GetOrCreateChild(httpPlatformElement, "environmentVariables");
+            var appBaseElement = envVariables.Elements("environmentVariable").SingleOrDefault(e =>
+                string.Equals((string)e.Attribute("name"), appBaseKeyName, StringComparison.CurrentCultureIgnoreCase));
+
+            if (appBaseElement == null)
```

Comment:
Yes https://github.com/aspnet/IISIntegration/pull/80/files#diff-2d39602621e11916e8439e3e56c8b466R180. Kind of... I just noticed it is missing the `[Fact]` attribute. The value of the `value` attribute needs to be set always to match how the application was published.